src/screens/settings/SettingsCustomCodeScreen/Components/SimpleCodeEditor.tsx

```
@@ -0,0 +1,591 @@
+          />
+        </Row>
+      ))}
+    </View>
+  );
+}
+
+export function SimpleCodeEditor({
+  highlightMode = 'combined',
+  onChangeText,
+  containerStyle,
+  onScroll,
```

ESLint

[error] 'onScroll' is defined but never used. Allowed unused args must match /^_/u.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'lines' is never reassigned. Use 'const' instead.

(prefer-const)